view.py

```diff
@@ -22,14 +27,14 @@ def run(self, c:controller.Controller, m:model):
         curses.noecho()
         curses.cbreak()
         curses.raw()
+        self.print_model(m=m)
         while True:
```

Comment:
This could be:
  while model.playing():

Reply:
We concluded that main() can have a loop:
  while model.playing():
    controller.wait()
    view.update()